lib/app/features/home/match_details/match_details_tabs/match_summary_tab/elements/halves_results_display.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:score_live/core/applocalization_context.dart';
+import 'package:score_live/models/live_match_model.dart';
+
+class FirstHalfResultDisplay extends StatelessWidget {
```

Comment:
Do you need two separated widgets instead of passing parameter `isHalfTime` ?

Reply:
done with boolean parameter as suggested